apps/wallet/src/ui/app/components/menu/content/Account.tsx

```diff
@@ -58,11 +63,36 @@ export function Account({ address }: AccountProps) {
                         leaveTo="transform opacity-0"
                     >
                         <Disclosure.Panel className="px-5 pb-4">
-                            <AccountActions accountAddress={address} />
+                            <AccountActions
+                                accountAddress={address}
+                                accountType={accountType}
+                            />
                         </Disclosure.Panel>
                     </Transition>
                 </div>
             )}
         </Disclosure>
     );
 }
+
+type AccountBadgeProps = {
+    accountType: AccountType;
+};
+
+function AccountBadge({ accountType }: AccountBadgeProps) {
+    switch (accountType) {
+        case AccountType.LEDGER:
+            return (
+                <div className="bg-gray-40 rounded-2xl border border-solid border-gray-45 py-1 px-1.5">
+                    <Text variant="captionSmallExtra" color="steel-dark">
+                        Ledger
+                    </Text>
+                </div>
+            );
+        case AccountType.DERIVED:
+        case AccountType.IMPORTED:
```

Comment:
Isn't there also a badge for imported accounts?

Reply:
there should be, I'll add one based off of https://www.figma.com/file/xu2VnyPAuruNi74UQzGigW/02-Wallet?node-id=6073%3A85759&t=694zeFjQm2XxuhN0-0